docs/future-work/oss-communication-improvements/build-a-tutorial.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Build a Tutorial
+rfc: not-required
+estimated-scope: medium
+improved-metric: community-engagement
+skills: video-production, writing, python
+output-type: tutorial
+status: evergreen
+---
+
+Create a tutorial with your experience with Monty to accomplish some goal.
+
+## Format Options
+
+The tutorial can be created in either format:
+- YouTube video
+- Blog post
+
+Both formats may potentially become core documentation if they add value to the existing getting started guides.
+
+
+## Potential Tutorial Topics
+
+- Running through the steps to set up Monty on a new configuration of hardware / OS.
+- How to add FLOPs measurements to a Monty experiment.
+- How to create a new dataset.
```

Comment:
We don't use datasets, we use environments. We should rephrase this. How would this be different from this tutorial: https://thousandbrainsproject.readme.io/docs/using-monty-in-a-custom-application ?

Reply:
Oh, I was thinking like the omniglot dataset or the compositional dataset Scott put together.  What is that data called?  Mesh models?

Reply:
Yeah those are the models that are then being rendered in the environment. Maybe you could rephrase it as "How to create new 3D objects and render then in the simulator" and "How to turn an existing dataset into a learning environment for Monty"